widgetastic_manageiq/__init__.py

```diff
@@ -4356,3 +4356,153 @@ def fill(self, value):
             return False
         self._all_item_elements[value or self.selected_item].click()
         return True
+
+class MigrationPlansList(Widget):
```

Comment:
Ignore this Widget, as this code is already being reviewed in the #7295 , which when merged, I will rebase this PR off-of master and make sure of correct commit history.